packages/components/FocusZone/src/FocusZone.types.ts

```diff
@@ -37,6 +37,27 @@ export type FocusZoneProps = React.PropsWithChildren<{
    */
   use2DNavigation?: boolean;
 
+  /**
+   * By default, pressing Tab within a FocusZone moves focus out of the FocusZone.
+   * This prop allows you to change that behavior.
+   *
+   * "None" : Default
+   * "NavigateWrap" : Tab allowed within FZ with circular navigation behavior
+   * "NavigateStopAtEnds" : Tab allowed within FZ and stops at ends
+   * "Normal" : Tab allowed within FZ, and focus is not trapped. Tabbing at the last element bring you out of the FZ.
+   *
+   * Recommended Usage - Moving focus OUT of the FocusZone - Custom Key Handlers
+   *      Example: Subscribe to key events. On the pressing of "Escape", move focus out of the FocusZone.
+   *
+   * Recommended Usage - Moving focus INTO the FocusZone
+   *    Let's take a CardList, for example. If a user is navigating through elements of the List, we don't want
+   *    focus to land within the FocusZone on initial focus. If it does, it will trap focus within the FocusZone and
+   *    ruin the user's workflow.
```

Comment:
Interesting, I think I fixed a similar issue by making the FZ itself focusable and its' own default tubbable element on macOS, should the user wish 😅

Reply:
Interesting - did you add a View as a slot to the FocusZone?